dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -331,34 +331,34 @@ BEGIN
     EDITTEXT 14008, 197, 103, 33, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
     CONTROL "", 14015, "Static", SS_OWNERDRAW, 70, 116, 100, 40
     CTEXT "Partiție %c", 14009, 95, 162, 50, 10
-    PUSHBUTTON "Curățare &disc…", 14010, 160, 159, 70, 15, WS_TABSTOP
+    PUSHBUTTON "Curăţare &disc", 14010, 160, 159, 70, 15, WS_TABSTOP
```

Comment:
This doesn't have any dots on romanian MS? That would be uncommon, as that is usually used by MS to indicate that the button will open a subdialog.

Reply:
No, the dots are not present in the Romanian translation. Please ignore the XP/2k3 typo. It is about "Curăţare &disc" and not about "Curăţire &disc". In the newer versions this typo is fixed.

![2024-03-28 01_13_40-Dialog - 1080](https://github.com/reactos/reactos/assets/113553683/9b5a246d-a4f9-442e-8c22-bf1d87dd4ca6)

Reply:
Ok great. Thank you for the confirmation pic! Resolved then.